Gemfile

```diff
@@ -19,12 +19,82 @@ source 'https://rubygems.org'
 
 gemspec :name => "chef-dk"
 
+# EXPERIMENTAL: ALL gems specified here will be installed in chef-dk omnibus.
+# This represents all gems that will be part of chef-dk.
+
+# All software we recognize needs to stay at the latest possible version. But
+# since that's not expressible here, we make it >= the last *known* version to
+# at least prevent downgrades beyond that:
+gem "appbundler", github: "chef/appbundler", branch: "jk/multiple-gems"
+gem "berkshelf"
 # Chef 12.8.1 Gem includes some extra files which can break gem installation on
 # windows. For now we are pulling chef from github at the tag as a workaround.
-gem "chef", github: "chef", tag: "v12.8.7"
+gem "chef", github: "chef/chef", branch: "v12.9.7"
+gem "cheffish", ">= 2.0.3"
+gem "chef-provisioning", ">= 1.6.0"
+gem "chef-provisioning-aws", ">= 1.8.0"
+gem "chef-provisioning-azure", ">= 0.5.0"
+gem "chef-provisioning-fog", ">= 0.18.0"
+gem "chef-provisioning-vagrant", ">= 0.11.0"
+gem "inspec", ">= 0.17.1"
+gem "ohai", ">= 8.13.0"
+gem "test-kitchen", ">= 1.6.0"
+
+# All of the following used to be software definitions we included:
```

Comment:
Might be helpfull to add a comment that clarifies why these gems don't have constraints except rubocop/jmespath